neo/NeoSystem.cs

```diff
@@ -34,10 +34,11 @@ public class NeoSystem : IDisposable
         public NeoSystem(Store store)
         {
             this.store = store;
+            Plugin.LoadPlugins(this);
             this.Blockchain = ActorSystem.ActorOf(Ledger.Blockchain.Props(this, store));
             this.LocalNode = ActorSystem.ActorOf(Network.P2P.LocalNode.Props(this));
             this.TaskManager = ActorSystem.ActorOf(Network.P2P.TaskManager.Props(this));
-            Plugin.LoadPlugins(this);
+            Plugin.NotifyPluginsLoadedAfterSystemConstructed();
```

Comment:
Does this construction runs sequentially?
Isn't the the ActorSystem.ActorOf something that can be still running in parallel?

Reply:
Those create the objects sequentially. Notice the call to `Props` happens before the call to `ActorOf`.

Reply:
My point was that in the `Props` call, which happens here first before the calls to `ActorOf`, the objects themselves are created and initialized, so even if `ActorSystem.ActorOf` did something asynchronously, the construction of the Blockchain, LocalNode, and TaskManager, would have already occurred before `Plugins.NotifyPluginsLoadedAfterSystemConstructed()` is called.

However, to fully answer your question. I looked inside the implementation of `ActorSystem.ActorOf`, and it appears it does not do asynchronous work here either.

Reply:
I got, Jeff, thanks.
The objects are already created and their constructor has already been called, right? Sounds sequential.

Reply:
Right.